### ISSUE TITLE
Who would I suggest to an addon author for help making it work with Cursebreaker?

### ISSUE THREAD

**Issue by Barleduq** [AcidWeb/CurseBreaker#355]
Hi! I made a comment on https://github.com/Acidfrost59/WOWBPCritterEmote saying that I was getting an error when I try to install using CurseBreaker.  The author said they'd be looking for help making that work.  I did point them to the bigwigs manager (as mentioned in a comment on this project a while ago.)  Is there some other place I can point people to who want more information on making this work? 

Thank you!
-Barleduq

**Comment by AcidWeb** [comment]
Nothing changed. Using bigwigs packager is a correct way: https://github.com/BigWigsMods/packager/wiki
Additionally in this case his manually created package don't work as it not contain directory with the addon code.

**Comment by Acidfrost59** [comment]
Hello AcidWeb
re: Acidfrost59/WOWBPCCritterEmote
I appreciated Barleduq's reaching out to you to see what I need to do to have my addon work with Cursebreaker. My problem is that I am not a programmer. I took over the addon to maintain it with the WOW version and to add new pets and zones. The only place I maintain this addon is on GitHub. That is the only repository, there are no externals. 

This is the .pkgmeta I created. Does it look correct?

package-as: CritterEmote

enable-nolib-creation: yes

externals:
  libs/WOWBPCritterEmote-1.0:
    url: https://github.com/Acidfrost59/WOWBPCritterEmote
    tag: master

manual-changelog:
    filename: CHANGELOG.txt

ignore:
  - README.txt
